pandas/core/series.py

```diff
@@ -391,6 +391,16 @@ def get_values(self):
         """ same as values (but handles sparseness conversions); is a view """
         return self._data.get_values()
 
+    def _repr_latex_(self):
```

Comment:
Can you actually remove this and move the `_repr_latex_` from `pandas/core/frame.py` and to `NDFrame` in `pandas/core/generic.py`?